node/src/main/scala/coop/rchain/node/api/package.scala

```diff
@@ -0,0 +1,69 @@
+package coop.rchain.node
+
+import cats.effect.Concurrent
+import cats.effect.concurrent.Semaphore
+
+import coop.rchain.blockstorage.BlockStore
+import coop.rchain.casper.MultiParentCasperRef.MultiParentCasperRef
+import coop.rchain.casper.SafetyOracle
+import coop.rchain.casper.protocol.CasperMessageGrpcMonix
+import coop.rchain.catscontrib._
+import coop.rchain.comm.discovery._
+import coop.rchain.comm.rp.Connect.ConnectionsCell
+import coop.rchain.grpc.{GrpcServer, Server}
+import coop.rchain.node.diagnostics.{JvmMetrics, NodeMetrics}
+import coop.rchain.node.model.diagnostics._
+import coop.rchain.node.model.repl._
+import coop.rchain.rholang.interpreter.Runtime
+import coop.rchain.shared._
+
+import io.grpc.netty.NettyServerBuilder
+import monix.eval.Task
+import monix.execution.Scheduler
+
+package object api {
+
+  // 16 MB is max message size allowed by HTTP2 RFC 7540
+  // grpc and netty can however work with bigger values
+  val maxMessageSize: Int = 16 * 1024 * 1024
```

Comment:
So we are restricted to 16 MB messages?

Reply:
Good catch @tymm ! We must use the gRPC max message size from the configuration (min 256KB, max 10MB and default 4MB). I don't know if these values are sensible. But that's how it is for now in the `Configuration`.